proposals-CG-WG.md

```diff
@@ -29,70 +29,73 @@ The purpose of Stage 0 proposals is to allow anyone to raise and explore the Pro
+    * [ ] A completed Working Draft for further iteration.
+    * [ ] Approval to transition the draft into a formal as the basis for their work as a [Working Draft](https://www.w3.org/policies/process/#RecsWD).
+
   
 # Stage 3: Implementation
 
 > NOTE: We don't yet have enough experience as we get towards Stage 3 and 4, so they aren't fully baked yet. It is possible that these
-> will change a lot as we learn, including potentially entirely replacing them with references to the W3C Process.
+> will change a lot as we learn, including potentially entirely replacing them with references to the W3C Process. 
 
-The purpose of Stage 3 Proposals is to increase implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR).
+The purpose of Stage 3 Proposals is to increase implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR). The spec should be considered finished, pending editorial nit review. However, since multiple implementations are expected in this stage, it is possible there will be further changes to normative content.
  
   * What's needed from champions?
-    * [ ] Proposal fully merged into the [Working Draft](https://www.w3.org/policies/process/#RecsWD)
-    * [ ] Web Platform Tests
-    * [ ] Further implementation experience
+    * [ ] Proposal fully merged into the [Working Draft](https://www.w3.org/policies/process/#RecsWD).
+    * [ ] Web Platform Tests are available.
+    * [ ] At least two independent implementers and no unresolved objections (where resolution may include an agreement not to address the issue).
   * What's asked of the **Working Group**? 
-    * [ ] Working Group consensus that the [Working Draft](https://www.w3.org/policies/process/#RecsWD) sufficiently resolves all of the issues raised at [Stage 2](#stage-2)
-    * [ ] Working Group consensus to publish the [Working Draft](https://www.w3.org/policies/process/#RecsWD) as the Working Group's [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR)
+    * [ ] Working Group consensus that the [Working Draft](https://www.w3.org/policies/process/#RecsWD) sufficiently resolves all of the issues raised at [Stage 2](#stage-2).
+    * [ ] Working Group consensus to publish the [Working Draft](https://www.w3.org/policies/process/#RecsWD) as the Working Group's [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR).
   
 # Stage 4: Publication
 
- The purpose of Stage 4 Proposals is to produce a [W3C Recommendation](https://www.w3.org/policies/process/#RecsW3C).
+ The purpose of Stage 4 Proposals is to produce a [W3C Recommendation](https://www.w3.org/policies/process/#RecsW3C). At this stage, the spec is merged and has finished editor review. This editor review could be lengthy, especially if the feature is large and/or the contributor is new to W3C process, but it will usually be short.
```

Comment:
```suggestion
 The purpose of Stage 4 Proposals is to produce a [W3C Recommendation](https://www.w3.org/policies/process/#RecsW3C). At this stage, the spec is merged and has finished editorial review. This editorial review could be lengthy, especially if the feature is large and/or the contributor is new to W3C process, but it will usually be short.
```